static/src/stylesheets/module/content-garnett/_content.scss

```diff
@@ -87,6 +87,11 @@
     }
 }
 
+.content__head {
```

Comment:
Should this be wrapped inside the `mq` as well?

Reply:
it could be, but it won't have any affect not in one i don't think?

Reply:
do you think it's unsafe to apply it blanketly?

Reply:
do you think 'blanketly' is a word?

Reply:
I honestisticly think it can have side effects yeah 😅 there are a couple of things with floats, margin collapse, etc. though I don't know if they apply here.

Reply:
👍 

added mq, just missed by GH diff

Reply:
me likey 👍